docs/quick-start-guide.md

```diff
@@ -111,11 +111,9 @@ import org.apache.spark.sql.CarbonContext
 * Create an instance of CarbonContext in the following manner :
 
 ```
-val cc = new CarbonContext(sc)
+val cc = new CarbonContext(sc, "<hdfs store path>")
```

Comment:
For Quick start : users always trying it at local machine, so they might create cc without "hdfs store path".  so please give two cases for creating cc:
Local machine :  val cc = new CarbonContext(sc, "local machine's store path")
HDFS: val cc = new CarbonContext(sc, "<hdfs store path>")